oauth2client/client.py

```diff
@@ -1617,6 +1617,23 @@ def _revoke(self, http_request):
         """
         self._do_revoke(http_request, self.access_token)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
+
+        Args:
+            blob: bytes, Message to be signed.
+
+        Returns:
+            tuple, A pair of the private key ID used to sign the blob and
+            the signed contents.
+        """
+        _abstract()
+
+    @property
+    def service_account_email(self):
+        """Get the email for the current service account."""
+        _abstract()
```

Comment:
Just `raise NotImplementedError()`. Consistency is often worthwhile but not in this specific case.